src/models/yamlProjectConfiguration.ts

```diff
@@ -312,6 +313,15 @@ export class YamlProjectConfiguration {
                 return true;
             }, "The day of the month and day of the week cannot be specified at the same time.");
 
+        // Used for a/b testing of genezio runtime.
+        // TODO: Remove after a/b testing is complete, and use a determined cloud provider.
+        const cloudProviderDefault =
+            ENVIRONMENT === "dev"
+                ? Math.random() < 0.5
+                    ? CloudProviderIdentifier.GENEZIO
+                    : CloudProviderIdentifier.CAPYBARA_LINUX
+                : CloudProviderIdentifier.GENEZIO;
+
```

Comment:
This approach has multiple disadvantages:
- Most of our configuration files are generated automatically and they already are setting the cloud provider to `genezio`, so this code will take effect rarely.
- This code will be executed for every genezio command, so for example, I can trigger a cloud provider change by running `genezio list` or `genezio local` which is not desirable.
- We don't want to change cloud provider for a project that is already deployed because currently we don't support changing providers on the fly.

I played around and come up with an approach on the deployment flow that solves the concerns above:
https://github.com/Genez-io/genezio/pull/818